test/RAST_SDK_server_test.pl

```diff
@@ -241,12 +241,23 @@ sub prepare_recent_old_genome {
         my $genome_ref = prepare_old_genome($assembly_ref, $genome_obj_name);
         test_reannotate_genome($genome_obj_name, $genome_ref);
     }, "test_reannotate_genome";
+
 lives_ok{
-        $genome_obj_name = "genome.4";
+        my $genome_obj_name = "genome.4";
         my $genome_ref = prepare_recent_old_genome($assembly_ref, $genome_obj_name);
         test_reannotate_genome($genome_obj_name, $genome_ref);
     }, 'test_reannotate_genome';
-done_testing(4);
+
```

Comment:
Maybe add some comments for this test and a TODO to switch to GFU in the future.  I wouldn't block on it though.